solutions/usaco-735.mdx

```diff
@@ -163,5 +163,48 @@ file_out.close()
 ```
 
 </PySection>
+	
+<JavaSection>
+	
+## Implementation
```

Comment:
this isn't your fault, but maybe some reformatting could be done on this file?
i think moving ## Implementation outside the languagesection (and deleting it from each of the cpp, py and java sections) would be a good idea, as well as moving the time complexity from the pysection outside